src/tests/ftest/util/server_utils.py

```diff
@@ -382,6 +383,13 @@ def clean_files(self, verbose=True):
             if cmd not in clean_cmds:
                 clean_cmds.append(cmd)
 
+            # Remove the shared memory segment associated with each io server
+            for srv_idx in range(NUM_IO_SERVERS):
+                key = 0x10242048 + srv_idx
```

Comment:
Magic values are frowned upon. It would be better to define the base key at the top of the file so that if necessary it can be imported from other modules. e.g. `D_TM_BASE_KEY=0x10242048`.

Reply:
This was a test run to verify the technique would work.  Now that I know it does what it was supposed to do, I'll work on an implementation that fits in.